base/sparse/cholmod.jl

```diff
@@ -1325,4 +1325,7 @@ function ishermitian(A::Sparse{Complex{Float64}})
     end
 end
 
+(*){Ti}(A::Symmetric{Float64,SparseMatrixCSC{Float64,Ti}}, B::SparseMatrixCSC{Float64,Ti}) = sparse(Sparse(A)*Sparse(B))
```

Comment:
I don't think our algorithm will be faster as such, but to use the CHOLMOD version we have to translate the matrices back and forth to `CHOLMOD.Sparse` matrices which adds a little overhead.